exotations/task_maps/task_map/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ add_definitions(-DEXOTICA_DEBUG_MODE)
 
 find_package(catkin REQUIRED COMPONENTS
   exotica
+  exotica_py
```

Comment:
That would be the way for pure CMake but it's rarely used in Catkin. We could do what you suggest by pulling the ```task_map_py``` into a separate package. It would be bit more tricky because ```libtask_map.so``` is a plugin library and would have to loaded as such, since you cannot link to a library and use it as a plugin in the same binary. This would then allow you remove/ignore both ```exotica_py``` and any extension wrappers.